### PR TITLE
Implements selectFindObject in MVP-611

### PR DESCRIPTION
**Pull request by interplanetarychris**
Note that the variable names in server.py should probably be changed to partial_string to match the intent of the search function selectFindObject() in database.py

### CODE REVIEW

**Comment by interplanetarychris** [comment]
Per conversion with @johngribbin, this endpoint was not in prior use, so I've updated it to map the parameter "partial_string" as to what gets passed to database.py

**Comment by interplanetarychris** [comment]
This appears to be made irrelevant by @johngribbin and @Kmoneal lack of awareness that it existed.  @Kmoneal has task to integrate/harmonize the changes needed for the SingleIOD entry form.

File: database.py [hunk 1]
```diff
@@ -3211,25 +3211,34 @@ def selectObjectInfluence_JSON(self, norad_num, fetch_row_count=100, offset_row_
     def selectFindObject(self, partial_string):
         """ Facilitate a search of objects contained in the database, by NORAD number or name.
         Provide a partial result of matches to allow the user to choose their specific object of interest.
+        Partial-match accepting IOD varient of the international designators
```
Comment (Kmoneal) [inline database.py]: variant not varient

File: database.py [hunk 2]
```diff
@@ -3211,25 +3211,34 @@ def selectObjectInfluence_JSON(self, norad_num, fetch_row_count=100, offset_row_
...
-                ORDER by norad_num ASC;
+                WHERE norad_num LIKE '%(PARTIAL)s%'
+                OR intl_desg LIKE %(PARTIAL)s
+                ORDER by norad_num ASC
+                LIMIT 100;
                 """
-        else:
+        except ValueError:
+            query_parameters = {'PARTIAL': '%' + partial_string + '%'}    # Append trailing wildcard to string
+            query_parameters['PARTIAL'] = query_parameters['PARTIAL'].replace("%20", "%")
```
Comment (Kmoneal) [inline database.py]: Add comment interpret space as wild card.

File: database.py (hunk 1 above)
Comment (Kmoneal) [inline database.py]: variant not varient

File: database.py (hunk 2 above)
Comment (Kmoneal) [inline database.py]: Add comment interpret space as wild card.

**Comment by Kmoneal** [comment]
@interplanetarychris ready to review.

**Pull request merged**